idaes/ui/fsvis/static/js/stream_table.js

```diff
@@ -35,7 +35,7 @@ export class StreamTable {
             let column_header = columns[col]
             if (column_header !== "") {
                 if (column_header === "Variable") {
-                    column_defs.push({headerName: column_header, field: column_header, filter: 'agTextColumnFilter', sortable: true, resizable: true});
+                    column_defs.push({headerName: column_header, field: column_header, filter: 'agTextColumnFilter', sortable: true, resizable: true, pinned: 'left',});
```

Comment:
Why not add a comment saying why this if clause exists?

Reply:
Added.